lib/main.dart

```diff
@@ -1,113 +1,151 @@
+      debugShowCheckedModeBanner: false,
+      title: 'Business Card',
+      home: Scaffold(
+        body: Container(
+          child: SafeArea(
+            top: false,
+            child: Container(
+              decoration: BoxDecoration(
+                image: DecorationImage(
+                    image: AssetImage(
+                      "assets/images/cardbackground.png",
+                    ),
+                    fit: BoxFit.fill),
+              ),
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: <Widget>[
+                  Container(
+                    decoration: BoxDecoration(
+                      shape: BoxShape.circle,
+                      boxShadow: [
+                        BoxShadow(
+                          blurRadius: Constants.avatarBlurShadow,
+                          color: Constants.avatarShadowColor,
+                          spreadRadius: 1,
+                        ),
+                      ],
+                    ),
+                    child: CircleAvatar(
+                      backgroundImage: AssetImage(
+                        'assets/images/me.jpg',
+                      ),
+                      radius: Constants.avatarRadius,
+                    ),
+                  ),
+                  SizedBox(
+                    height: Constants.avatarAndPersonNameSizeBoxHeight,
+                  ),
+                  Text(
+                    'Kevin Guillen',
+                    style: TextStyle(
+                      fontSize: Constants.personNameFontSize,
+                      fontWeight: FontWeight.bold,
+                      color: Constants.personNameTextColor,
+                      letterSpacing: Constants.letterSpacing,
+                    ),
+                  ),
+                  SizedBox(
+                    height: Constants.personNameAndPositionSizeBoxHeight,
+                  ),
+                  Text(
+                    'Flutter Developer Trainee',
+                    style: TextStyle(
+                      fontSize: Constants.positionFontSize,
+                      fontWeight: FontWeight.bold,
+                      color: Constants.positionTextColor,
+                      letterSpacing: Constants.letterSpacing,
+                    ),
+                  ),
+                  Column(
+                    children: [
+                      Divider(
+                        height: 60,
+                        thickness: 3,
+                        color: Colors.blueAccent.shade100,
+                        indent: 60,
+                        endIndent: 60,
+                      ),
+                    ],
+                  ),
+                  Card(
+                    color: Constants.cardsBackgroundColor,
+                    child: ListTile(
+                      leading: Padding(
+                        padding: EdgeInsets.only(
+                          top: 8.0,
```

Comment:
This could be a constant like mediumPadding ?

Reply:
fixed